packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -37,12 +37,12 @@ function isJsonPayload (value: SignerPayloadJSON | SignerPayloadRaw): value is S
 
 const POPUP_WINDOW_OPTS: chrome.windows.CreateData = {
   focused: true,
-  height: 600,
+  height: 640,
   left: 150,
   state: 'normal',
   top: 150,
-  type: 'popup',
-  width: 360
+  type: 'panel',
```

Comment:
Isn't `panel` experimental? Doesn't it make the popup stick to bottom-right corner or something like that?

Reply:
good catch, I think I experimented a little and it must be a relic :see_no_evil: